### PR TITLE
[mesheryctl] Create dedicated section for tokens in default config

### LINKED ISSUE #2435: [mesheryctl] Token location should be in section dedicated to tokens
#### Description
The following is an example of the v0.5.0-beta-4 default context, created by mesheryctl:

```
contexts:
  local:
    endpoint: http://localhost:9081
    token:
      name: Default
      location: auth.json
    platform: docker
    adapters:
    - meshery-istio
    - meshery-linkerd
    - meshery-consul
    - meshery-octarine
    - meshery-nsm
    - meshery-kuma
    - meshery-cpx
    - meshery-osm
    - meshery-traefik-mesh
    channel: stable
current-context: local
```

#### Expected Behavior
`      location: auth.json` should be in a dedicated `token` section of the meshconfig, like so:
```
contexts:
   <context1-name>:
	endpoint: <url to meshery server rest api>
	token: <name of token variable in this config file>
platform: <type of platform: ”docker” or “kubernetes”> 
# Future: specify type of kubernetes (e.g. eks)
adapters: <collection of names of service mesh adapters: “istio”,“linkerd”,”consul”,”nginx-sm”,”octarine”,”tanzu-sm”,”citrix-sm”,”kuma”,”osm”,”nsm”> 
# Future: ”app-mesh”,”traefik-mesh”
   <context2-name>:
	endpoint: <url to meshery server rest api>
	token: <name of token variable in this config file>
platform: <type of platform: ”docker” or “kubernetes”>
current-context: <context name>
tokens:
- name: <token1-name>
  location: <token-location>
- name: <token2-name>
  value: <token-value> 
  # Future: allow embedding of token certificate
```

See "[meshery config template](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.2oq40t5wa4bu)" section of the Meshery CLI & Documentation doc for more details.

Tokens are to be independently controlled outside of the lifecycle of a given context.

Token name is to remain under the named context section to provide a "foreign key" to the token referenced in the token section.

```
    token:
      name: Default
```

#### Environment:
- Host OS: 
- Meshery Version: 
- Kubernetes Version: 
- Browser: 

<!-- Optional

### PR DESCRIPTION
**Pull request by aneeshnema**
**Description**

This PR fixes #2435 

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/pkg/utils/TestConfig.yaml [hunk 1]
```diff
@@ -1,23 +1,23 @@
 contexts:
   local:
     endpoint: http://localhost:9081
-    token:
-      name: Default
-      location: auth.json
+    token: Default
```
Comment (leecalcote) [inline mesheryctl/pkg/utils/TestConfig.yaml:4]: @aneeshnema I missed this before. How is the testconfig.yaml used? Do unit tests make use of this?

**Review by leecalcote — APPROVED** [review]
@aneeshnema this looks great. Were you able to successfully run `mesheryctl system config <platform> -t` and `mesheryctl perf -f <file> -t`  as tests?

File: mesheryctl/pkg/utils/TestConfig.yaml (hunk 1 above)
Comment (aneeshnema) [reply mesheryctl/pkg/utils/TestConfig.yaml:4]: Yes, the `TestViewCmd` unit test inside `channel_test.go` uses it.

Comment (leecalcote) [reply mesheryctl/pkg/utils/TestConfig.yaml:4]: Wonderful. Unit tests in-action!

**Review by leecalcote — APPROVED** [review]
Good stuff, @aneeshnema 👍

**Pull request merged**